examples/powertools-examples-idempotency/src/main/java/helloworld/App.java

```diff
@@ -52,7 +56,7 @@ public App(DynamoDbClient client) {
      *     <li>Second call (and next ones) will retrieve from the cache (if cache is enabled, which is by default) or from the store, the handler won't be called. Until the expiration happens (by default 1 hour).</li>
      * </ul>
      */
-    @Idempotent // *** THE MAGIC IS HERE ***
+    @Idempotent
```

Comment:
I liked my comment, let's keep a bit of fun...

Reply:
Magic re-added